src/webgl/context-state.js

```diff
@@ -317,25 +318,25 @@ const GL_STATE = {
   [GL.UNPACK_ALIGNMENT]: {
     type: GLint,
     params: GL.UNPACK_ALIGNMENT,
-    setter: (gl, value) => gl.pixelStorei(value)
+    setter: (gl, value) => gl.pixelStorei(GL.UNPACK_ALIGNMENT, value)
```

Comment:
I missed the first case..... :( since no one is using it...I guess I'll just commit but not hurry to publish it